fs-core/main/default/lwc/formulaShareRulesListView/formulaShareRulesListView.js

```diff
@@ -372,30 +382,32 @@ export default class TreeGrid extends NavigationMixin(LightningElement) {
         refreshApex(this.provisionedValue);
     }
 
-
     // Action method to update a rule to active/inactive
     spinnerClasses;
     activateDeactivate(row, actionName) {
+        console.log('>>>row: ' + JSON.stringify(row, null, '\t'));
         const rowDeveloperName = row['developerName'];
+        console.log('#1');
```

Comment:
I noticed you added quite a few console.log statements, but it would be good to avoid outputting anything for production. Are you OK to either remove or comment out?

Reply:
Removed.